lib/dbs.js

```diff
@@ -79,7 +79,7 @@ module.exports = {
     "old": new Mysql({
         host: "db_old",
         user: "root",
-        password: "abc123",
+        password: "",
```

Comment:
I believe this change is just because your DB was copied from the old repo, which didn't use a password. Like the docker-compose.yml change, this can vary depending on each of our environments. Let's not bother committing changes to it.